tests/tracing/bug01732-001.phpt

```diff
@@ -0,0 +1,56 @@
+--TEST--
+Test for bug #1732 - Trace flamegraph format output for time
+--INI--
+xdebug.mode=trace
+xdebug.start_with_request=0
```

Comment:
That should be `=no` to be consistent with the [documentation](https://xdebug.org/docs/all_settings#start_with_request).

Reply:
I copy-pasted this from another test, but OK.